### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by Sven-Lyco**
fixed typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,33 +6,36 @@
 
 ## FIRST GIT WORKLOW
```
Comment (macfrei) [inline README.md:7]: Maybe add a hint that this is locally on your machine?

File: README.md [hunk 2]
```diff
@@ -6,33 +6,36 @@
...
 
-- `cd <fodlername>` : navigate to where you want to make a new folder
+- `cd <foldername>` : navigate to where you want to make a new folder
 - `take <foldername>` : create a new folder and navigate in it (`take` is a combi of `mkdir` and `cd`)
 - `mkdir <foldername>`: create new folders you need
 - `touch <filename>` : create new files you need
 - `touch README.md`: create a `README.md` to describe your project
 - `git init` : initiate git in your first created folder, when all files and folders are created
```
Comment (macfrei) [inline README.md:14]: It is *really*  important to already be in the project folder

File: README.md [hunk 3]
```diff
@@ -6,33 +6,36 @@
...
 - `touch .gitignore` : create a `.gitignore` - file
 - `ls -la` : look for hidden files, you dont want to uplaod to Git / Github
-- `echo '.DS_Store' >> .gitignore`: write filenames in `.gitignore`
-- `git add .` : add everything to the Stage
-- `git commit -m "Initial commit"` : initial commit
+- `echo '.DS_Store' >> .gitignore`: write filenames and folders in `.gitignore`
+- `git add -A` : add everything to the Stage (Stage is a pool for commits)
+- `git commit -m "initial commit"` : initial commit
```
Comment (macfrei) [inline README.md:19]: Commit messages should start with a capital letter.
```suggestion
- `git commit -m "Initial commit"` : initial commit
```

File: README.md [hunk 4]
```diff
@@ -6,33 +6,36 @@
...
 
 ## FIRST GITHUB-WORKFLOW
```
Comment (macfrei) [inline README.md:21]: Maybe also here it would be best to add that this is "remote" and not local :)

File: README.md [hunk 5]
```diff
@@ -6,33 +6,36 @@
...
 
-- Add Repository on GITHUB
+- Create a new Repository on GITHUB
```
Comment (macfrei) [inline README.md:23]:
```suggestion
- Create a new Repository on [GITHUB](https://github.com/)
```
:octocat:

File: README.md [hunk 6]
```diff
@@ -6,33 +6,36 @@
...
   - repository-name = local-folder-name
-  - next Page: Quicksetup
+  - choose public or private
```
Comment (macfrei) [inline README.md:25]: I would recommend to always have them public :)

File: README.md [hunk 7]
```diff
@@ -6,33 +6,36 @@
...
+  - **don't** initialize this repository
```
Comment (macfrei) [inline README.md:26]:
```suggestion
  - **don't** initialize this repository with a readme or gitignore
```

File: README.md [hunk 8]
```diff
@@ -6,33 +6,36 @@
...
+- next Page: Quicksetup
   - choose SSH
-  - copy and paste link for push in Terminal: `git remote add origin git@github.com:USERNAME/REPOSITORY.git`
+  - copy and paste first link for **push** in Terminal: `git remote add origin git@github.com:USERNAME/REPOSITORY.git`
```
Comment (macfrei) [inline README.md:29]: This is not a "push". This only connects our local repository with our remote repository. The "push" comes later with `git push -u origin main`. 🙂

**Review by macfrei — COMMENTED** [review]
Very nice! Julia and I looked over it and we added some comments you might want to consider 😎 👍

File: README.md (hunk 8 above)
Comment (Sven-Lyco) [reply README.md:29]: very good hint! :)

**Pull request merged**